include/llama/mapping/AoS.hpp

```diff
@@ -10,17 +10,18 @@ namespace llama::mapping
     /// Array of struct mapping. Used to create a \ref View via \ref allocView.
     /// \tparam AlignAndPad If true, padding bytes are inserted to guarantee that struct members are properly aligned.
     /// If false, struct members are tighly packed.
-    /// \tparam LinearizeArrayDimsFunctor Defines how the array dimensions should be mapped into linear numbers and
+    /// \tparam T_LinearizeArrayDimsFunctor Defines how the array dimensions should be mapped into linear numbers and
```

Comment:
I'm a bit confused by the naming here. Our guidelines mandate a `T` prefix (not `T_`. Did you change this yourself or is the clang-tidy file wrong?

Reply:
I have not yet applied the full clang-tidy from CRP. Especially the renamings are missing. So this is consistent with how I named some of the old template parameters.